blazor/maps/polygon.md

```diff
@@ -91,4 +91,174 @@ The following example shows how to customize the polygon shape over the geometry
     }
 }
 ```
-![Blazor Maps with polygon shape](./images/Polygon/blazor-map-polygon-shape-on-maps-layer.png)
\ No newline at end of file
+![Blazor Maps with polygon shape](./images/Polygon/blazor-map-polygon-shape-on-maps-layer.png)
+
+## Tooltip for polygon shape
+
+Tooltip is used to display more information about a polygon shape on mouse over or touch end event. This can be enabled by setting the [`Visible`]() property to **true** in the  [`MapsPolygonTooltipSettings`](). Additonally, you need to set the [`TooltipText`]() property in the [`MapsPolygon`](../api/maps/polygonSettingsModel) to show the tooltip text separately for each polygon shape, if you add 'n' numbers of polygon shapes.
+
+### Tooltip customization
+
+The following properties are available in the [`MapsPolygonTooltipSettings`]() to customize the tooltip of the Maps component.
+
+* [`Fill`]() - Applies the color of the tooltip in the polygon shape.
```

Comment:
* [`Fill`]() - Applies the color of the tooltip in the polygon shape.
* [`BorderColor`]() - To customize the color of the border of the tooltip in the polygon shape.
* [`BorderWidth`]() - To customize the width of the border of the tooltip in the polygon shape.
* [`BorderOpacity`]() - To customize the opacity of the border of the tooltip in the polygon shape.
* [`FontFamily`]() - To customize the font family of the text in the tooltip of the polygon shape.
* [`FontWeight`]() - TTo customize the font weight of the text in the tooltip of the polygon shape.
* [`FontStyle`]() - To customize the font style of the text in the tooltip of the polygon shape.
* [`FontColor`]() - To customize the font color of the text in the tooltip of the polygon shape.
* [`FontSize`]() - To customize the font size of the text in the tooltip of the polygon shape.
* [`FontOpacity`]() - TTo customize the opacity of the text in the tooltip of the polygon shape.